index.html

```diff
@@ -998,16 +998,112 @@
 
 </div>
 
-## Safety Systems {#safety}
+## Safety, Abuse, and Fraud {#safety}
+
+There are some cases where privacy and safety are in tension. Many of these cases are
+short-term, only existing until we develop a system that achieves both privacy and
+safety. Long-term cases are rare, but still occasionally come up. We can distinguish
+three categories:
+
+1. A person's privacy exists in a tradeoff with their own safety.
```

Comment:
Actually, a publisher's privacy impacts public safety, not just their own safety. Publishers who uses anonymity to avoid accountability for publishing disinformation or to implement fraud cause significant public harm.